lib/WebpackOptionsDefaulter.js

```diff
@@ -85,14 +85,15 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 		this.set("output.filename", "[name].js");
 		this.set("output.chunkFilename", "make", options => {
 			const filename = options.output.filename;
-			if (typeof filename === "function") return filename;
-			const hasName = filename.includes("[name]");
-			const hasId = filename.includes("[id]");
-			const hasChunkHash = filename.includes("[chunkhash]");
-			// Anything changing depending on chunk is fine
-			if (hasChunkHash || hasName || hasId) return filename;
-			// Elsewise prefix "[id]." in front of the basename to make it changing
-			return filename.replace(/(^|\/)([^/]*(?:\?|$))/, "$1[id].$2");
+			if (typeof filename !== "function") {
+				const hasName = filename.includes("[name]");
+				const hasId = filename.includes("[id]");
+				const hasChunkHash = filename.includes("[chunkhash]");
+				// Anything changing depending on chunk is fine
+				if (hasChunkHash || hasName || hasId) return filename;
+				// Elsewise prefix "[id]." in front of the basename to make it changing
+				return filename.replace(/(^|\/)([^/]*(?:\?|$))/, "$1[id].$2");
+			}
```

Comment:
You need to return something in the case of filename being a function too.

Maybe just return `[id].js`